yarn.lock

```diff
@@ -4649,18 +4627,6 @@
     "@leafygreen-ui/polymorphic" "^1.2.0"
     "@leafygreen-ui/tokens" "^2.0.1"
 
-"@leafygreen-ui/typography@^16.1.0":
```

Comment:
not sure why some LeafyGreen dependencies were affected but when I upgraded a completely different library the same thing happened so maybe this is just out of date 🤷